src/main/scala/scalismo/package.scala

```diff
@@ -38,6 +38,10 @@ package object scalismo {
       gc.SetScheduleTime(60, TimeUnit.SECONDS)
       gc.Start()
 
+      // we need to stop the garbage collection when the JVM shutdowns.
+      // Otherwise it prevents the JVM from shutting down correctly.
+      Runtime.getRuntime().addShutdownHook(new Thread { gc.Stop() })
```

Comment:
@clangguth Good points.  We could have an optional argument to initialize, which allows the user to specify the interval. By setting it to 0 we could disable our own garbage collection. In this way, a user who wants to use the original VTK auto-GC could at least disable our GC mechanism. 

I will adapt my code accordingly and update the PR.